.github/workflows/dhis2-verify-app.yml

```diff
@@ -101,54 +101,45 @@ jobs:
             - name: Test
               run: yarn test
 
-    #e2e:
-    #    runs-on: ubuntu-latest
-    #    if: "!contains(github.event.head_commit.message, '[skip ci]')"
-    #    needs: [install]
-    #
-    #    strategy:
-    #        matrix:
-    #            containers: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
-    #
-    #    steps:
-    #        - name: Checkout
-    #          uses: actions/checkout@v2
-    #
-    #        - uses: actions/setup-node@v1
-    #          with:
-    #              node-version: 12.x
-    #
-    #        - uses: actions/cache@v2
-    #          id: yarn-cache
-    #          with:
-    #              path: '**/node_modules'
-    #              key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-    #
-    #        - name: Install Cypress binary
-    #          run: yarn cypress install
-    #
-    #        - name: End-to-End tests
-    #          uses: cypress-io/github-action@v2
-    #          with:
-    #              install: false
-    #              record: true
-    #              parallel: true
-    #              start: ${{ env.SERVER_START_CMD }}
-    #              wait-on: ${{ env.SERVER_URL }}
-    #              wait-on-timeout: 300
-    #              cache-key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-    #              group: 'e2e'
-    #              tag: ${{ github.event_name }}
-    #          env:
-    #              CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
-    #              COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
-    #              STORYBOOK_TESTING: true
-    #              SERVER_START_CMD: 'yarn cy:server'
-    #              SERVER_URL: 'http://localhost:5001'
+    e2e:
+        runs-on: ubuntu-latest
+        if: "!github.event.push.repository.fork && github.actor != 'dependabot[bot]'"
+        needs: [install]
+
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
+
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 12.x
+
+            - uses: actions/cache@v2
+              id: yarn-cache
+              with:
+                  path: '**/node_modules'
+                  key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+
+            - name: Install Cypress binary
+              run: yarn cypress install
+
```

Comment:
Perhaps we don't need to do this either?

```suggestion
```

Reply:
Yup, removed it.